# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xaralx/wx.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

at startup/new document wxFindWindowAtPointer() crashes here because of the
delayed window setup in wxwidgets

--- XaraLX-trunk/wxOil/cursor.cpp	(revision 1750)
+++ XaraLX-trunk/wxOil/cursor.cpp	(working copy)
@@ -584,11 +584,11 @@
 	// this stops the cursor being hijacked when we pop.
 	wxWindow*	pRenderWnd = DocView::GetCurrentRenderWindow();
 	wxPoint		ptDontCare;
-	wxWindow*	pWndAtPtr  = wxFindWindowAtPointer( ptDontCare );
+	//wxWindow*	pWndAtPtr  = wxFindWindowAtPointer( ptDontCare );
 	if( NULL != pRenderWnd )
 	{
-		bool	fDrawCtrl = NULL != pWndAtPtr ? pWndAtPtr->IsKindOf( CLASSINFO(wxCamDrawControl) ) : false;
-		if( !fOnlyRendWnd || fDrawCtrl || pWndAtPtr == pRenderWnd )
+		//bool	fDrawCtrl = NULL != pWndAtPtr ? pWndAtPtr->IsKindOf( CLASSINFO(wxCamDrawControl) ) : false;
+		if( !fOnlyRendWnd /*|| fDrawCtrl || pWndAtPtr == pRenderWnd*/ )
 			wxSetCursor(hCursor);
 		else
 			wxSetCursor( *wxSTANDARD_CURSOR );
